integration_tests/transform/ecmascript-modules-support/package.json

```diff
@@ -0,0 +1,14 @@
+{
+  "dependencies": {
+    "babel-preset-env": "^1.6.1"
```

Comment:
we have a dependency on `babel-preset-env` in the root already, so this is not needed https://github.com/facebook/jest/blob/1521c516c2cae8f4bfe41605640f273f9c17fb87/package.json#L18